requirements.txt

```diff
@@ -9,3 +9,5 @@ paramiko>=2.1.2
 yarn-api-client>=0.2.3
 pexpect>=4.2.0
 pycrypto>=2.6.1
+pytest>=3.3.2
```

Comment:
I thought test dependencies were supposed to go into `requirements-test.txt`.  Is there a compelling reason for having these here?  They will end up getting loaded as part of the general pip installs otherwise.

Reply:
Will move dependency changes to test requirements..thanks for letting me know.